docs/SetupAndTeardown.md

```diff
@@ -127,11 +127,11 @@ describe('Scoped / Nested block', () => {
 // 1 - afterAll
 ```
 
-## Order of execution of describe and test blocks
+## describe 和 test 块的执行顺序
 
-Jest executes all describe handlers in a test file _before_ it executes any of the actual tests. This is another reason to do setup and teardown inside `before*` and `after*` handlers rather than inside the describe blocks. Once the describe blocks are complete, by default Jest runs all the tests serially in the order they were encountered in the collection phase, waiting for each to finish and be tidied up before moving on.
+Jest 会在所有真正的测试开始之前先执行测试文件里所有的 describe 处理程序（handlers）。 这是在 `before*` 和 `after*` 处理程序中而不是在describe块中执行setup和teardown的另一个原因。 一旦describe块完成，默认情况下Jest会按照在收集阶段遇到的顺序运行所有测试，等待每个测试完成并整理好，然后再继续。
```

Comment:
1. (handlers) 可删除。
2. setup 与 teardown请翻译
3. This is another reason to do setup and teardown inside `before*` and `after*` handlers rather than inside the describe blocks.  这块译得不准确